package.json

```diff
@@ -2,6 +2,13 @@
   "name": "ui-we-want-to-sleep",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "test": "react-scripts test",
```

Comment:
This should be replaced with `jest`. How to test React/TS apps with Vite is explained [here](https://codingwithmanny.medium.com/quick-jest-setup-with-vitejs-react-typescript-82f325e4323f), for example.